projects/sudoers/build.sh

```diff
@@ -15,25 +15,70 @@
 #
 ################################################################################
 
-if [ $SANITIZER == "address" ]; then
-    export LDFLAGS="-fsanitize=address"
-elif [ $SANITIZER == "undefined" ]; then
-    export LDFLAGS="-fsanitize=undefined"
-elif [ $SANITIZER == "coverage" ]; then
-    export LDFLAGS="$CFLAGS"
+# Debugging
+env
+
+# Move ASAN-specific flags into ASAN_CFLAGS and ASAN_LDFLAGS
+# That way they don't affect configure but will get used when building.
+if [ $SANITIZER == "coverage" ]; then
+    export ASAN_CFLAGS="$COVERAGE_FLAGS"
```

Comment:
nit: Maybe name this variable `OSS_FUZZ_CFLAGS` instead of `ASAN_CFLAGS` since it won't make sense if you are using UBSAN/MSAN (or even coverage for that matter).

Reply:
These are the names that sudo uses in its Makefiles when configured with --enable-asan.  I'm re-using what is already available in sudo.

Reply:
Ah. didn't realize this.